modules/post/windows/gather/credentials/smartermail.rb

```diff
@@ -106,12 +138,56 @@ def get_smartermail_creds(path)
     end
 
     username = data.match(/<sysAdminUserName>(.+)<\/sysAdminUserName>/)
-    password = data.match(/<sysAdminPassword>(.+)<\/sysAdminPassword>/)
+    password = data.scan(/<(sysAdminPassword|sysAdminPasswordHash)>(.+)<\/(sysAdminPassword|sysAdminPasswordHash)>/).flatten[1]
+
     result['username'] = username[1] unless username.nil?
-    result['password'] = decrypt_des(Rex::Text.decode_base64(password[1])) unless password.nil?
+
+    if password
+      begin
+        result['password'] = decrypt_des(Rex::Text.decode_base64(password))
+        result['password_type'] = :password
+      rescue OpenSSL::Cipher::CipherError
+        result['password'] = password
+        result['password_type'] = :nonreplayable_hash
```

Comment:
Yeah should be a String instead of a Symbol, and then this value will be used by ```:private_type```